content/posts/guia-de-bolso-do-dev-front-end.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Guia de bolso do Desenvolvedor Front-End'
+authors: Fabiano de Lima Abreu
+type: post
+date: 2018-02-20
+excerpt: 'Lista de referencia de sites para consulta e estudos que todo front-end deve conhecer. '
+categories:
+  - Geral
+  - Mercado e Comportamento
+---
+
+Este artigo é destinado para você iniciante que está a procura de uma boa lista de referências, mas tenho certeza que esta lista de referências será util pra você seja qual o seu nivel de conhecimento.
+
+Este artigo é destinado para você iniciante que está a procura de uma boa lista de referências, mas tenho certeza que esta lista de referências será util pra você seja qual o seu nivel de conhecimento.
+Procurei coletar a melhor lista possivel de links que sejam úteis para o dia a dia do DEV, e que se faz importante ter um fácil acesso.
+Este artigo é destinado para você que está a procura de uma boa lista de referências independente de qual seu nivel de conhecimento.
```

Comment:
Frase redundante. Você iniciou o artigo falando exatamente isso. Acho que mantém só esta frase no início.

Reply:
Frase redundante removida.